### ISSUE TITLE
Nothing show to enable Asiapay_Pdcptb in admin Panel

### ISSUE THREAD

**Issue by NangVisibee** [asiapay-lib/asiapay-Magento-2.x#9]
I can't find to enable Asiapay_Pdcptb in admin Panel. 
Store -> Configurations -> Advanced -> Advanced
Don't have "Advanced" step in admin Panel. How can I configure for it?
I put folder under ../app/code/ and enable module but can't find Store -> Configurations -> Advanced -> Advanced.

**Comment by juliusponce-apph** [comment]
From Magento 2.2.* Now Advanced Menu is no longer available.
Go to the admin side of the store and log-in, then go to _**System** -> **Web Setup Wizard** -> **Module Manager**_
Ensure that the component name  '**Asiapay_Pdcptb**' is enabled in the action tab.

**Comment by NangVisibee** [comment]
Now. I can see to configure . Payment can be shown only normal checkout page. It can't shown on multishipping checkout page.
How can i configure for it?
@juliusponce-apph

**Comment by SimonLeungAPHK** [comment]
@NangVisibee Please contact our sales team and IT team for more support.

**Comment by NangVisibee** [comment]
Yes.  I sent email to 'service@paydollar.com'. But no one reply to me.
